packaging/convoy-agent/launch

```diff
@@ -58,6 +57,7 @@ volume_agent_glusterfs() {
 }
 
 volume_agent_glusterfs_internal() {
+    wait_for_metadata
```

Comment:
Why did you add this? Is it needed for the refactoring you did around separating out convoy + the agent? 
The reason I _didnt_ have it was that volume_agent_glusterfs_internal is called by volume_agent_glusterfs and it is the one that waits on metadata.